app/task_routes.py

```diff
@@ -0,0 +1,111 @@
+from os import abort
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+from datetime import datetime
+import requests  
+import os
+
+tasks_bp = Blueprint("task", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    if "description" not in request_body:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+    if "title" not in request_body:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
```

Comment:
Since the return statements are identical, let's combine these together:
```python
    if "description" not in request_body or "title" not in request_body:
        return make_response(jsonify({"details": "Invalid data"}), 400)
```